spek-artwork.patch

```diff
@@ -0,0 +1,30 @@
+diff --git a/src/spek-artwork.cc b/src/spek-artwork.cc
+index cd7e78c..322555a 100644
+--- a/src/spek-artwork.cc
++++ b/src/spek-artwork.cc
+@@ -2,6 +2,7 @@
+ #include <wx/iconbndl.h>
+ 
+ #include "spek-artwork.h"
++#include <iostream>
+ 
+ class SpekArtProvider : public wxArtProvider
+ {
+@@ -16,7 +17,7 @@ wxBitmap SpekArtProvider::CreateBitmap(
+ 	(void) id, (void) client, (void) size;
+ #ifdef OS_UNIX
+     if (id == ART_SPEK) {
+-        return wxArtProvider::GetBitmap("spek", client, size);
++        return wxArtProvider::GetBitmap("cc.spek.Spek", client, size);
```

Comment:
you can add `copy-icon: true` in the manifest and drop this patch. It will make the icon-rename copy them instead of just renaming.